pkg/controller/servicecatalog/provisionedproduct/lifecycle.go

```diff
@@ -24,7 +23,10 @@ const (
 	msgProvisionedProductStatusSdkPlanInProgress = "provisioned product is awaiting plan approval"
 	msgProvisionedProductStatusSdkError          = "provisioned product has status ERROR"
 
-	errUpdatePending = "Provisioned product is already under change, not updating"
+	errUpdatePending                        = "provisioned product is already under change, not updating"
```

Comment:
Just for my education, what is the purpose of having messages as strings when they are used only once? I saw this pattern in the python code and I'm curios about it.
Or, are they used later in the tests to validate and test output against the expected message?
Thanks!